client/api_key_project_assignment.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+type ApiKeyRole string
+
+const (
+	ApiKeyRoleDeployer ApiKeyRole = "Deployer"
+	ApiKeyRoleViewer   ApiKeyRole = "Viewer"
+	ApiKeyRolePlanner  ApiKeyRole = "Planner"
+	ApiKeyRoleAdmin    ApiKeyRole = "Admin"
+)
```

Comment:
we've got these also defined in `client/team_project_assignment.go`
Those are the same roles a regular user/team have. Can we reuse them?

Reply:
Yes.